libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -670,6 +676,15 @@ void AP_TECS::_detect_underspeed(void)
     }
 }
 
+void AP_TECS::_detect_overspeed(void)
+{
+    if (_flags.overspeed) {
+        _flags.overspeed = _TAS_state > _TASmax * 0.5;
```

Comment:
I don't think this is a good way to unset the overspeed flag.
Consider the following configuration:
- `AIRSPEED_MAX` = 25
- `AIRSPEED_MIN` = 20
- and cruise somewhere in between.

So basically you need to stall in order to clear your overspeed flag.

Reply:
> I don't think this is a good way to unset the overspeed flag. Consider the following configuration:
> 
>     * `AIRSPEED_MAX` = 25
> 
>     * `AIRSPEED_MIN` = 20
> 
>     * and cruise somewhere in between.
> 
> 
> So basically you need to stall in order to clear your overspeed flag.

Yes, thanks.  This was Ryan, Bob, Michelle and myself mucking around to see if we could get the thing to not over-speed :-)  May have gotten a little over-enthusiastic there...

Clamping into airspeed max yields oscillation, as you might imagine.  0.9 was no better.  0.5 moved things along nicely ;-)